components/login.jsx

```diff
@@ -0,0 +1,112 @@
+var React = require('react');
+var Router = require('react-router');
+var Link = Router.Link;
+
+var teachAPI = require('../lib/teach-api');
+
+var Login = React.createClass({
+  getDefaultProps: function() {
+    return {
+      teachAPI: teachAPI,
+      alert: defaultAlert
```

Comment:
Oh it's fine! I just happen to do this on my phone. Sorry about that (my train is delayed lol)